keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -161,6 +247,15 @@ uid=1000(alice) gid=1000(alice) groups=1000(alice),50000(group-in-image),60000
 
 ## Motivation
 
+<!--
```

Comment:
You don't need to keep these comments in the file - it just makes it appear longer :)